pyglmnet/pyglmnet.py

```diff
@@ -498,6 +499,8 @@ def __init__(self, distr='poisson', alpha=0.5,
         self.solver = solver
         self.learning_rate = learning_rate
         self.max_iter = max_iter
+        self.n_iter = 0
+        self.converged = False
```

Comment:
It's usually not good practice to add too many attributes to an object unless it's really necessary. I would say if you raise a warning, that is good enough. Is there a use case to record this or the number of iterations somewhere?

Reply:
We use both (in our private fork) to programmatically check that `fit()` actually converged in a reasonable way. Warnings are fine for interactive human users, but automation needs something stored with the object, particularly within GLMCV where callbacks aren't available.

Reply:
I see, okay fair enough. However, you don't need `self.converged` because you can find that based on what is the `max_iter` you passed. And I would add an `underscore` at the end to indicate that it's a parameter that was not passed: so `self.n_iter_` rather than `self.n_iter`